src/components/forms/newListingForm.jsx

```diff
@@ -115,6 +121,33 @@ class NewListingForm extends Component{
 
         );
     }
+    
+    submitForm = () => {
+        console.log("made it to submitform")
+        this.handleSubmit(this.state)
+        this.setState(this.initialState)
+    }
+
+    handleSubmit = character => {
+        this.makePostCall(character).then(callResult => {
+          if (callResult.status === 201) {
+            this.setState({ characters: [...this.state.characters, callResult.data] });
+          }
+        });
+    }
```

Comment:
Be sure that the status matches with the backend so that there are no issues.

Reply:
yep